docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -94,9 +94,108 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+Installation of generated systemd unit files.
+
+Podman generated unit files include an `[Install]` section, which carries installation information for the unit. It is used by the enable and disable commands of the systemctl(1) during installation.
+
+Once you have generated the systemd unit file. Copy the generated systemd file to ```/usr/lib/systemd/system``` for installing as a root user and to ```/usr/lib/systemd/user``` for installing it as a non-root user.
+
+Enable the copied one or more units. This will create a set of symlinks, as encoded in the "[Install]" sections of the indicated unit files.
+
+Note: Coping the unit to ```/usr/lib/systemd/system``` and enabling it marks the unit to be automatically started at boot.
+
+```
+# Copy and enable all the generated systemd files.
+
+$ sudo cp pod-systemd-pod.service container-great_payne.service /usr/lib/systemd/system
+$ systemctl enable pod-systemd-po.service
+Created symlink /etc/systemd/system/multi-user.target.wants/pod-systemd-po.service → /usr/lib/systemd/system/pod-systemd-po.service.
+Created symlink /etc/systemd/system/default.target.wants/pod-systemd-po.service → /usr/lib/systemd/system/pod-systemd-po.service.
+$ systemctl is-enabled pod-systemd-po.service
+enabled
+```
+
+Generated .service file can be used to control the status of the pod and the underlying containers once they are enabled. Since these service files where not generated with `--new` operations with systemctl will modify the same pod|container.
```

Comment:
Sorry, missed these earlier.  "Generated .service" -> "The generated .service"  also "where not generated" -> "were not generated"